lib/chat_web/live/main_live/index.html.heex

```diff
@@ -739,7 +749,11 @@
                       <button
                         class="mr-4 flex items-center"
                         phx-click={
-                          if @room_requests == [], do: nil, else: show_modal("room-request-list")
+                          cond do
+                            @db_status.writable == :no -> show_modal("restrict-write-actions")
+                            @room_requests == [] -> nil
+                            true -> show_modal("room-request-list")
+                          end
```

Comment:
we need to figure out another way to do this
This becomes too much of logic

Reply:
might be creating `<.room_request_button ...>`

Reply:
done